mlflow/sparkml.py

```diff
@@ -0,0 +1,172 @@
+"""
```

Comment:
@tomasatdatabricks Is `mllib.py` a better name for this module? @aarondav Pointed out that the current name is very similar to `spark.py`